_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any `img` element that is [visible][] except if the `src` [attribute value][] references an SVG document.
+
+## Expectation
+
+The image resource referenced by the `src` [attribute value][] of each target element either does not contain text expressing anything in a [human language][] or it is [essential][] that the text is rendered with that specific presentation.
+
+## Assumptions
+
+The text is the most significant content in the image. If there is text in the image, but it is not the most significant content, this rule might fail but the success criterion might still be satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.5: Images of Text][sc1.4.5]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element references an image resource that does not contain text.
+
+```html
+<img src="/test-assets/shared/fireworks.jpg" alt="fireworks going off behind the Eiffel tower at night" />
+```
+
+#### Passed Example 2
+
+This `img` element references an image resource that contains text but where the presentation of the text is essential to convey the information.
+
+```html
+<p>The following image illustrates the use of cursive writing in the late nineteenth century.</p>
+<img src="/test-assets/0va7u6/letter.jpg" alt="A letter written in 1894 showing the use of cursive writing" />
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `img` element references an image resource that contains text and the way the text is presented is not relevant.
+
+```html
+<img
+	src="/test-assets/0va7u6/textimage.jpg"
+	alt="The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test rules."
+/>
+```
```

Comment:
My first reaction when seeing this example was "Wow, the SC does forbid screenshots?" Seeing the image and reading the SC, I think it doesn't since the presentation can be essential in a screenshot (but not in that image).

…which comes to the point of whether we should clear that upfront with a screenshot Passed Example. Typically instruction pages like https://www.howtogeek.com/school/using-windows-admin-tools-like-a-pro/lesson5/ (OK, not the first image in that page…) with screenshots to show precisely where to click. These seem to pass the SC, I think.

Reply:
Interesting... that image is not a screenshot :-) I created an image of text, instead of capturing a screenshoot

A screenshot such as the ones in the page you mention seems to me to fall under the assumption. The SC applies to instances where images of text are used to convey information. Such images are the ones where text was rendered in a way to achieve a particular "visual effect" unless the picture contains other "significant visual content". 

The screenshot:
- Conveys information
- Has text rendered to achieve a particular visual effect, although the visual effect here is not about the styling of the text, but the relative placement of text within the user interface (and I have doubts about the SC applying to this)
- Has other significant visual content (or else it would be a text interface and not a graphical user interface and the tutorial would be about the command line)

What do you think @Jym77 ?

Reply:
Yes, I think I agree that the text in this kind on screenshot is not the most significant information 👍